lib/libc/minimal/Kconfig

```diff
@@ -14,29 +14,30 @@ config MINIMAL_LIBC_NON_REENTRANT_FUNCTIONS
 	  option may require an additional memory protection region.
 
 config MINIMAL_LIBC_MALLOC
-	bool "Minimal libc malloc implementation"
+	bool "Legacy alias for COMMON_LIBC_MALLOC"
 	default y
+	select COMMON_LIBC_MALLOC
 	help
 	  Enable the minimal libc's implementation of malloc, free, and realloc.
 	  Disable if you wish to provide your own implementations of these functions.
 
 config MINIMAL_LIBC_MALLOC_ARENA_SIZE
-	int "Size of the minimal libc malloc arena"
+	int "Legacy alias for COMMON_LIBC_MALLOC_ARENA_SIZE"
```

Comment:
Prpbably let's just select proper value for `COMMON_LIBC_MALLOC_ARENA_SIZE` based on `MINIMAL_LIBC_MALLOC_ARENA_SIZE`

Reply:
I stuck this logic into the source code as I couldn't figure out a way to do it in Kconfig -- if `COMMON_LIBC_MALLOC_ARENA_SIZE` is left at the default value of zero, then the code will use the value of `CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE`.